networking/src/Node/Message/Class.hs

```diff
@@ -45,6 +50,22 @@ class Message m where
     default formatMessage :: F.Buildable m => m -> T.Text
     formatMessage = F.sformat F.build
 
+instance Message Void where
+    messageCode _ = 0
+    formatMessage _ = "Void"
+
+instance Message UpdateVote where
```

Comment:
I was a little concerned about placing these particular Message instances in the module along with the typeclass definition. I ended up having to do this, as opposed to how I typically couple the instances with their type definitions, because these types are from core and core cannot depend on networking since networking already depends on core.

If anyone has a better idea for how I should do this, please let me know!